lib/qr-bills.rb

```diff
@@ -2,10 +2,11 @@
 require 'qr-bills/qr-exceptions'
 require 'qr-bills/qr-params'
 require 'qr-bills/qr-html-layout'
+require 'qr-bills/qr-prawn-layout'
 require 'qr-bills/qr-creditor-reference'
 
 module QRBills
-  def self.generate(qr_params)
+  def self.generate(qr_params, pdf = nil)
```

Comment:
thanks for the PR! I would find better to pass an additional params directly in the ```qr_params``` variable, you can add a new parameter there called ```[:output][:prawn][:pdf]```, so we can leave all prawn objects under that place. Or eventually we can create a new section for the inputs like "[:format_inputs][:prawn][:pdf]"